web-client/src/App.tsx

```diff
@@ -1,13 +1,22 @@
 import { BrowserRouter } from 'react-router-dom';
 import MainLayout from 'components/MainLayout';
 import AppRoutes from 'routes/index';
+import {IntlProvider} from 'react-intl';
+import messages from 'intl/index';
+import  {useSettingsStore}  from 'store/useSettings';
+
+const App = () => {
+const {language} = useSettingsStore();
```

Comment:
do you have eslint enabled (and prettier added) in intellj or visual studio code?

Reply:
Yes